packages/design-system/src/components/button.tsx

```diff
@@ -18,6 +18,7 @@ const colors = [
   "positive",
   "neutral",
   "ghost",
+  "dark",
 ] as const;
```

Comment:
Need to add it here too https://github.com/webstudio-is/webstudio-designer/blob/9726664bd8dd0f4c62f9605f075bf9564e2abff0/packages/design-system/src/components/button.stories.tsx#L17-L23